tests/system/providers/google/cloud/automl/example_automl_dataset.py

```diff
@@ -0,0 +1,199 @@
+        task_id="create_bucket",
+        bucket_name=DATA_SAMPLE_GCS_BUCKET_NAME,
+        storage_class="REGIONAL",
+        location=GCP_AUTOML_LOCATION,
+    )
+
+    upload_file = LocalFilesystemToGCSOperator(
+        task_id="upload_file_to_bucket",
+        src=CSV_FILE_LOCAL_PATH,
+        dst=DATA_SAMPLE_GCS_OBJECT_NAME,
+        bucket=DATA_SAMPLE_GCS_BUCKET_NAME,
+    )
+
+    # [START howto_operator_automl_create_dataset]
+    create_dataset = AutoMLCreateDatasetOperator(
+        task_id="create_dataset",
+        dataset=DATASET,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    dataset_id = create_dataset.output['dataset_id']
+    # [END howto_operator_automl_create_dataset]
+
+    # [START howto_operator_automl_import_data]
+    import_dataset_task = AutoMLImportDataOperator(
+        task_id="import_dataset_task",
+        dataset_id=dataset_id,
+        location=GCP_AUTOML_LOCATION,
+        input_config=IMPORT_INPUT_CONFIG,
+    )
+    # [END howto_operator_automl_import_data]
+
+    # [START howto_operator_automl_specs]
+    list_tables_spec_task = AutoMLTablesListTableSpecsOperator(
+        task_id="list_tables_spec_task",
+        dataset_id=dataset_id,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_automl_specs]
+
+    # [START howto_operator_automl_column_specs]
+    list_columns_spec_task = AutoMLTablesListColumnSpecsOperator(
+        task_id="list_columns_spec_task",
+        dataset_id=dataset_id,
+        table_spec_id="{{ extract_object_id(task_instance.xcom_pull('list_tables_spec_task')[0]) }}",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_automl_column_specs]
+
+    # [START howto_operator_automl_update_dataset]
+    update = deepcopy(DATASET)
+    update["name"] = '{{ task_instance.xcom_pull("create_dataset")["name"] }}'
+    update["tables_dataset_metadata"][  # type: ignore
+        "target_column_spec_id"
+    ] = "{{ get_target_column_spec(task_instance.xcom_pull('list_columns_spec_task'), target) }}"
+
+    update_dataset_task = AutoMLTablesUpdateDatasetOperator(
+        task_id="update_dataset_task",
+        dataset=update,
+        location=GCP_AUTOML_LOCATION,
+    )
+    # [END howto_operator_automl_update_dataset]
+
+    # [START howto_operator_list_dataset]
+    list_datasets_task = AutoMLListDatasetOperator(
+        task_id="list_datasets_task",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_list_dataset]
+
+    # [START howto_operator_delete_dataset]
+    delete_dataset = AutoMLDeleteDatasetOperator(
+        task_id="delete_dataset",
+        dataset_id="{{ task_instance.xcom_pull('list_datasets_task', key='dataset_id_list') | list }}",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+        trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
Nit: Keep trigger rule outside of the documentation markers (START / END)

Reply:
@bhirsz I have deleted it